Soomla/Assets/Plugins/Soomla/Profile/ProfileEvents.cs

```diff
@@ -621,10 +771,22 @@ public void onInviteFailed(String message)
 
 		public static Action<Provider, string> OnInviteCancelled = delegate {};
 
+		public static Action<Provider, bool, string> OnGetLeaderboardsStarted = delegate {};
```

Comment:
@EugeneButusov @Blinkq we won't be merging ValueObject in this release, but this doesn't prevent us from starting using ValueObject in other callbacks. 
Actually I think it's even better to merge it, but comment out the new callbacks, so that we keep the old ones for now but have all the classes ready. 
And @EugeneButusov can already make these lines use ValueObject